camayoc/types/settings.py

```diff
@@ -155,7 +163,6 @@ class Configuration(BaseModel):
     camayoc: CamayocOptions
     quipucords_server: QuipucordsServerOptions
     quipucords_cli: QuipucordsCLIOptions
-    openshift: list[OpenShiftOptions]
```

Comment:
Since `OpenShiftOptions` is no longer needed, please remove that class definition from this file.